infra/apim_v2/api/billing-portal-api/v1/openapi_merge/openapi_merge.json

```diff
@@ -7,7 +7,7 @@
       "inputFile": "./openapi_template.json"
     },
     {
-      "inputFile": "../../../../../../src/main/resources/swagger/api-docs.json",
+      "inputFile": "../../../../../../../docs/pagopa/selfcare-onboarding-backend/openapi.json",
```

Comment:
it might be better to avoid exposing bff (backend-for-frontend) as external API, its nature should be to prepare data for `onboarding-frontend` in this case....you could expose `/onboarding/checkRecipientCode` by apim directly